docs/Supported-.editorconfig-options.md

```diff
@@ -1,5 +1,7 @@
-The dotnet-format global tool supports the core set of EditorConfig options*:
+# Supported .editorconfig options
+The dotnet-format global tool supports the core set of [EditorConfig options](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties)* as well as the [.NET coding convention settings for EditorConfig](https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2019)**.
```

Comment:
Thought it was better to simplify this file and point users to the vs editorconfig docs.